app/models/grda_warehouse/hud/client.rb

```diff
@@ -1297,23 +1292,19 @@ def presented_with_family?(after: nil, before: nil)
       raise 'After required if before specified.' if before.present? && ! after.present?
 
       hh = if before.present? && after.present?
-        recent_households = households.select do |_, entries|
+        households.select do |_, entries|
```

Comment:
Making a note for @eanders to review this in code.

Reply:
✅ I've reset all the stuff around this change and just disabled rubocop for the relevant lines.